packages/gatsby/src/commands/build-html.ts

```diff
@@ -14,6 +15,30 @@ import { IProgram, Stage } from "./types"
 type IActivity = any // TODO
 type IWorkerPool = any // TODO
 
+export interface IWebpackWatchingPauseResume extends webpack.Watching {
+  suspend: () => void
+  resume: () => void
+}
+
+let devssrWebpackCompiler: webpack.Compiler
+let devssrWebpackWatcher: IWebpackWatchingPauseResume
+let needToRecompileSSRBundle: boolean = true
```

Comment:
```suggestion
let needToRecompileSSRBundle = true
```

lint fails because:
```
25:5   error    Type boolean trivially inferred from a boolean literal, remove type annotation  @typescript-eslint/no-inferrable-types
```
(which is weird rule to have, but ok :) )